server/server.go

```diff
@@ -1331,20 +1323,7 @@ func (s *Server) loadModules(ctx context.Context, txn storage.Transaction) (map[
 func (s *Server) getCompiler() *ast.Compiler {
 	s.mtx.RLock()
```

Comment:
Do we still need the server mtx? If not, let's remove it.

Reply:
removed mtx